### ISSUE TITLE
Actionsheet: Focus is not cycling back to first and last button.

### ISSUE THREAD

**Issue by pankajpandey01** [SAP/ui5-webcomponents-react#3868]
**Describe the bug**
Focus is not moving to first button when down arrow key is pressed on last button
Focus is not moving to last button when up arrow key is pressed on first button

**Isolated Example**
https://sap.github.io/ui5-webcomponents-react/?path=/docs/modals-popovers-actionsheet--default-story

**To Reproduce**
1. Go to https://sap.github.io/ui5-webcomponents-react/?path=/docs/modals-popovers-actionsheet--default-story
2. Press down arrow key on last button
3. Press up arrow key on first button

**Expected behavior**
 on press down arrow key on last button, focus should move to first button
 on press up arrow key on first button, focus should move to last button

UI5 menu button has expected behavior: https://ui5.sap.com/#/entity/sap.m.MenuButton/sample/sap.m.sample.MenuButton

**UI5 Web Components for React Information**
`@ui5/webcomponents-react` version: 1.6.0
Operating System: All
Browser: All

**Stakeholder**
Organization: {SuccessFactors}
Business impact: {a11y issue}

**Comment by Lukas742** [comment]
Hi @pankajpandey01 

the arrow-key navigation doesn't loop for the SAPUI5 ActionSheet neither (see [here](https://ui5.sap.com/#/entity/sap.m.ActionSheet/sample/sap.m.sample.ActionSheet)). Since I also find it a bit strange that the menu navigation runs in a loop but that of the ActionSheet does not, I will contact our accessibility experts and ask if this is the intended behavior.

**Comment by Lukas742** [comment]
Hi @pankajpandey01 

I just received an answer from our accessibility expert:

> In principle, the circular navigation of the focus has advantages and disadvantages. That is why it is unfortunately not implemented uniformly, not even in the operating systems. While predictable action is preferably implemented on macOS, it is stopped at the end in the menu. On Windows, people decided decades ago on circular navigation to be able to quickly navigate from the first entry (default action) to the last entry (the properties).
> 
> Advantages: The user can quickly jump from the beginning to the end without having to move their fingers much (because the user already has their finger on the cursor pad). Little additional motor effort, but poorly operable if the screen is not being watched at the same time.
> 
> Disadvantages: The user cannot steer specifically to the end by arbitrarily pressing the arrow keys and stop there. Little additional motor effort for HOME/END, but predictable stop at the end, regardless of whether the screen is being watched or not. 
> 
> In our designs we are keen to recommend predictable navigation, i.e. circular navigation is usually not recommended.
> Targeted jumps, in circularly acting controls, would be feasible via HOME and END.
> Our design recommendations are more focused on predictability, since we have many users who are proficient in ten-finger keyboard operation and often are not watching the screen because data entry is occurring.

Considering this, the current arrow navigation is correct, but we are missing some other navigation options like HOME and END. We will [look into](https://github.com/SAP/ui5-webcomponents-react/issues/3875) that.

For the Menu, it might even be possible that the cycle-behavior is changed again.
